src/astUtils/reflection.ts

```diff
@@ -19,160 +18,166 @@ export function isXmlFile(file: (BrsFile | XmlFile)): file is XmlFile {
 
 // Statements reflection
 
-export function isBody(statement: Statement): statement is Body {
+export function isBody(statement: any): statement is Body {
```

Comment:
Changed these parameters to the `any` type, since we might pass other object types into them. It _could_ be `Statement | Expression`, but since we're checking for `.constructor.name`, might as well just let developers pass in any object.

Reply:
I was bitten by this `any` when converting my code - it is definitely a risk of bugs IMHO to not narrow to `Statement | Expression`.

Reply:
Sure. I guess `Statement`s and `Expression`s are the most common use case. I do see valid reasons for passing in any variable to determine whether it's a statement or an expression. However, in those cases the variable can be cast as `any`.

Reply:
done.